client/pluginManger.go

```diff
@@ -104,20 +106,24 @@ func (p *PluginManager) GetPlugins() (pluginList *InstalledPluginList, err error
 	return
 }
 
-func getPluginsInstallQuery(names []string) string {
+func (p *PluginManager) getPluginsInstallQuery(names []string) string {
 	pluginNames := make([]string, 0)
 	for _, name := range names {
 		if name == "" {
 			continue
 		}
+		if strings.Contains(name, "@") {
+			p.installPluginWithVersion(name)
```

Comment:
Function `getPluginsInstallQuery` is only expected to get a query string array. We should not do more things. It'll be more difficult to maintain if it has too many powers. So my suggestion is that do it outside of this function.